src/bvh/patch.hpp

```diff
@@ -41,81 +41,13 @@
 #include <vector>
 #include "snapshot.hpp"
 
-#ifdef BVH_ENABLE_KOKKOS
 #include <Kokkos_Core.hpp>
-#endif
 
 namespace bvh
 {
 #if 0
```

Comment:
Hmm, we probably should have removed this dead code a long time ago

Reply:
Removed the `#if 0` block completely.